merlin/examples/workflows/feature_demo/feature_demo.yaml

```diff
@@ -17,92 +17,116 @@ env:
         HELLO: $(SCRIPTS)/hello_world.py
         FEATURES: $(SCRIPTS)/features.json
 
```

Comment:
i would recommend we just do this for one of the steps instead of all of them

Reply:
Okay. I updated so that it only uses yaml anchors for the first step.